container-images/cann/Containerfile

```diff
@@ -0,0 +1,35 @@
+# Base image with CANN for compilation
+ARG ASCEND_VERSION=8.0.rc2.beta1-910b-openeuler22.03-py3.10
+
+FROM quay.io/ascend/cann:${ASCEND_VERSION} AS builder
+ARG GOLANG_VERSION
+COPY ../scripts /scripts
+RUN export http_proxy=http://172.17.0.1:7890 && https_proxy=http://172.17.0.1:7890 && chmod +x /scripts/*.sh && \
+    cann_in_sys_path=/usr/local/Ascend/ascend-toolkit; \
+    cann_in_user_path=$HOME/Ascend/ascend-toolkit; \
+    if [ -f "${cann_in_sys_path}/set_env.sh" ]; then \
+        source ${cann_in_sys_path}/set_env.sh; \
+        export LD_LIBRARY_PATH=${cann_in_sys_path}/latest/lib64:${cann_in_sys_path}/latest/aarch64-linux/devlib:${LD_LIBRARY_PATH} ; \
```

Comment:
We should consider supporting both x86 and arm

Reply:
If it's easy this is a good idea.